cloud/pubnub_utils.js

```diff
@@ -0,0 +1,48 @@
+var PubNub = require('cloud/pubnub');
+
+module.exports.sendMessage = sendMessage;
+module.exports.replayChannel = replayChannel;
+
+// Returns a Promise. 
+// See https://parse.com/docs/cloudcode/guide#cloud-code-advanced
+function sendMessage(pubkey, subkey, channel, message, successCallback, errorCallback) {
+    // return Parse.Cloud.httpRequest({
+    //     url: getSendMessageUrl(pubkey, subkey, channel, message)
+    // });
+    var pubnub = PubNub({
+        publish_key: pubkey,
+        subscribe_key: subkey
+    });
+
+    pubnub.publish({
+        channel: channel,
+        message: message,
+        callback: function (result) {
```

Comment:
No need to wrap the callbacks here. Right now you can change this to callback: successCallback. If we end up changing it to response, it will be
callback:response.success,
error:response.error